app/assets/stylesheets/modules/_landing.scss

```diff
@@ -0,0 +1,109 @@
+.hero {
+  background-color: $white;
+
+  &-title {
+    color: $dark-gray;
+    margin-bottom: em(5);
+    text-align: center;
+  }
+
+  &-subtitle {
+    font-style: italic;
+    font-weight: 300;
+    margin-top: em(5);
+    text-align: center;
+  }
+
+  &-main-text {
+    padding: 0 0 em(50);
+    text-align: center;
+    @include span-columns(8 of 12);
+    @include shift(2);
+  }
+
+}
+
+.organization {
+  background-color: $base-background-color;
+
+  &-title {
+    color: $white;
+    text-align: center;
+  }
+
+  &-subtitle {
+    color: $white;
+    font-weight: 400;
+    padding-bottom: 1em;
+    text-align: center;
+
+    & a {
+      border-bottom: 1px solid $white;
+      color: $white;
+      display: inline-block;
+      line-height: 0.97em;
+      margin: 0.2em auto;
+      text-shadow:/* stroke around*/
```

Comment:
Colon after property should be followed by one space<br>Declaration should not have a space before the terminating semicolon